client/orderbook/orderbook_test.go

```diff
@@ -588,9 +611,18 @@ func TestOrderBookUnbook(t *testing.T) {
 				make([]*cachedOrderNote, 0),
 				true,
 			),
-			note:     makeUnbookOrderNote(5, "ob", [32]byte{'d'}),
-			expected: nil,
-			wantErr:  true,
+			note: makeUnbookOrderNote(5, "ob", [32]byte{'b'}),
+			expected: makeOrderBook(
+				2,
+				"ob",
+				[]*Order{
+					makeOrder([32]byte{'b'}, msgjson.SellOrderNum, 10, 1, 2),
+					makeOrder([32]byte{'c'}, msgjson.SellOrderNum, 10, 2, 5),
+				},
+				make([]*cachedOrderNote, 0),
+				true,
+			),
+			wantErr: false,
```

Comment:
And this "Unbook sell order with future sequence value" test seems to have worked because it was failing due to unknown id error, not the one the name claims to test, note, I changed order ID `d`->`b`

Reply:
Fixed other similar issues (expectation mismatch) I found in these tests too.